iris/meta.yaml

```diff
@@ -39,7 +41,9 @@ requirements:
         - pyke
         - udunits2
         - cf_units
-        - libmo_unpack  # [not win]
+        - mo_pack  # [not win]
+        - nc_time_axis
+        - iris-grib  # [not win]
```

Comment:
I guess this will fail b/c the package was named `iris_grib` intead of `iris-grib`.